README.md

```diff
@@ -57,7 +60,7 @@ def duplicates?(array)
 end
 ```
 
-Snippet 5 - Big O:
+Snippet 5 - Big O: **O(n^2)** ; nested each loops; if either array gets longer, the time increases exponentially
```

Comment:
Your logic is correct here, but the efficiency should be O(n * m) to indicate that there are two different arrays (as you indicated in your explanation)